file-formats/suco/type/zif_aff_suco_v1.intf.abap

```diff
@@ -0,0 +1,44 @@
+"! AFF Type for SUCO (Authorization Default Variant)
+INTERFACE zif_aff_suco_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Leading Application</p>
+    "! Leading application
+    BEGIN OF ty_leading_application,
+      "! <p class="shorttext">Program ID</p>
+      "! Program ID
+      "! $default 'R3TR'
+      program_id   TYPE c LENGTH 4,
+      "! <p class="shorttext">Object Type</p>
+      "! Object type
+      "! $required
+      object_type  TYPE c LENGTH 4,
+      "! <p class="shorttext">Object Name</p>
+      "! Object name
+      "! $required
+      object_name  TYPE c LENGTH 40,
+      "! <p class="shorttext">Service Type</p>
+      "! Service type
+      service_type TYPE c LENGTH 16,
+      "! <p class="shorttext">Service Name</p>
+      "! Service name
+      service_name TYPE string,
+    END OF ty_leading_application.
+
+  TYPES:
+    "! <p class="shorttext">Object Type SUCO (Authorization Default Variant)</p>
+    "! Object type SUCO (authorization default variant)
+    BEGIN OF  ty_main,
+      "! $required
+      format_version      TYPE zif_aff_types_v1=>ty_format_version,
```

Comment:
Is my understanding correct, that you support ABAP language version "default" and "cloudDevelopment" but *not* "keyUser"? If so, this change should be done to avoid the selection of "keyUser".

```suggestion
      format_version      TYPE zif_aff_types_v1=>ty_format_version_cloud,
```

Reply:
Sorry, this was a wrong suggestion!